CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Contributing to GitHub Analyzer](#contributing-to-github-analyzer)
+  - [Code of Conduct](#code-of-conduct)
+  - [Questions](#questions)
+  - [Filing a bug or feature](#filing-a-bug-or-feature)
+  - [Submitting changes](#submitting-changes)
+    - [Sample Workflow](#sample-workflow)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing to GitHub Analyzer
+
+Thank you for contributing to GitHub Analyzer! Below you can find some core
+guidelines for contributing to the project.
+
+## Code of Conduct
+
+Be kind and respectful to the members of the community. Take time to educate
+others who are seeking help. Harassment of any kind will not be tolerated.
+
+## Questions
+
+If you have questions or ideas that are not a good fit for an issue
+(i.e., not a feature request or bug) feel free to use the repository's [discussions](https://github.com/crashappsec/github-analyzer/discussions) to get feedback from the community and maintainers.
+
+## Filing a bug or feature
+
+1. When filing a bug or request, please check existing issues to see if there
+   exist some issue capturing the same topic already.
+
+1. When filing bugs, please provide as detailed steps of reproduction as possible.
```

Comment:
should we create github issue template with these?

Reply:
Yeah no harm doing so